### DISCUSSION TITLE
UI gets messed up after using a custom blade component

### DISCUSSION THREAD

**Discussion by gerizoo** [filamentphp/filament#13131] [Help] [answered]
### Package

Form builder

### Package Version

v3.2

### How can we help you?

So I made a custom blade component to be used in a create form. However, when I add the component to the form, the UI gets messed up. For example the navigation bar isn't collapsible anymore and the color of the create button disappears. But when i hover over the create button then it appears like it should be. 
(See images below)

Without component:
<img width="1675" alt="Screenshot 2024-06-07 at 12 15 27" src="https://github.com/filamentphp/filament/assets/31694846/16fa87f3-2f50-4d69-ab15-624fde4a52ba">

With:
<img width="1527" alt="Screenshot 2024-06-07 at 12 14 53" src="https://github.com/filamentphp/filament/assets/31694846/5a24980d-6822-4638-9b60-d4c651d0eacc">

**Code snippets** 

Component:
<img width="656" alt="Screenshot 2024-06-07 at 12 23 13" src="https://github.com/filamentphp/filament/assets/31694846/ea9f943d-4461-4535-96c3-9919b5c63560">

Component class:
<img width="426" alt="Screenshot 2024-06-07 at 12 32 20" src="https://github.com/filamentphp/filament/assets/31694846/c03b5082-d793-45f8-ad93-ead411d45aa9">

Form:
<img width="383" alt="Screenshot 2024-06-07 at 12 14 30" src="https://github.com/filamentphp/filament/assets/31694846/9945e3e6-03f1-49a6-8673-803b221e96ff">

**Notice**: it is a blade component from Bladewind UI. Not sure if that could be an issue.

**Comment by leandrocfe** [comment, marked as answer]
It might not be an issue, but I have a suggestion: https://filamentphp.com/plugins/awcodes-shout

**Reply by gerizoo** [reply]
Exactly what I need, thank you! I'll still keep the discussion open so Dan can see it, in case it's a bug.
